src/test/java/ReceiptAnalysisTest.java

```diff
@@ -102,6 +102,16 @@
   private static final Optional<Long> TIMESTAMP_IN_1900S =
       Optional.of(Long.valueOf(INSTANT_IN_1900S.toEpochMilli()));
 
+  private static final String RAW_TEXT_WITH_PRICE = "the price is $12.77 in total";
+  private static final String RAW_TEXT_WITH_PRICE_NO_DOLLAR_SIGN = "the price is 12.77 in total";
+  private static final String RAW_TEXT_WITH_MULTIPLE_PRICES =
+      "the items cost $8.99, $2.79, and $1.99, so the total is $12.77 after the $1.00 discount";
```

Comment:
I'm am of the opinion that if it's used once in a test, you should create it in the test. This way the list of global variables doesn't get too long to read.